now-mainnet.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
We can remove this file since we won't be using Vercel to deploy the mainnet build.